Tests/RxCocoaTests/DelegateProxyTest.swift

```diff
@@ -415,6 +415,7 @@ extension DelegateProxyTest {
 
             _ = (control as! NSObject).rx.deallocated.subscribe(onNext: { _ in
                 deallocated = true
+                expectation.fulfill()
```

Comment:
WKWebView is not immediately deallocated for some reason. We need to wait until this happens. The timeout on the `wait` will make sure that the test won't run forever